pkg/engine/update_test.go

```diff
@@ -0,0 +1,40 @@
+package engine
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShortenPolicyPackPath(t *testing.T) {
+	tests := []struct {
+		path     string
+		expected string
+	}{
+		{
+			path:     "/Users/username/test-policy",
+			expected: "/Users/username/test-policy",
+		},
+		{
+			path:     "./..//test-policy",
+			expected: "../test-policy",
+		},
+		{
+			path:     "/Users/username/averylongpath/one/two/three/four/five/six/seven/eight/nine/ten/eleven/twelve/test-policy",
+			expected: "/Users/.../twelve/test-policy",
+		},
+		{
+			path:     "nonrootdir/username/averylongpath/one/two/three/four/five/six/seven/eight/nine/ten/eleven/twelve/test-policy",
+			expected: "nonrootdir/username/.../twelve/test-policy",
+		},
+		{
+			path:     "C:Documents and Settings/username/My Documents/averylongpath/one/two/three/four/five/six/seven/eight/test-policy",
```

Comment:
If we want a more representative example for a Windows file path, we should use backslashes, right? (Though depending on who is providing the data, it could be the case that in-practice we are always passed a path that already has the separators canonicalized to forward slashes.)

Also, should we write `C:\Documents and Settings\...` too? Or is the `C:Documents` (without the separator) intentional?

Reply:
Changed the test to use backslashes!